sdk/storage/azure-storage-blob-cryptography/src/main/java/com/azure/storage/blob/specialized/cryptography/EncryptedBlobClientBuilder.java

```diff
@@ -915,4 +921,17 @@ public EncryptedBlobClientBuilder requiresEncryption(boolean requiresEncryption)
         this.requiresEncryption = requiresEncryption;
         return this;
     }
+
+    /**
+     * Sets the partition chunk size
+     * @param gcmEncryptionRegionLength the region length used for encrypting the blob
```

Comment:
We should make sure to update the documentation to:
1. Align the terminology to authenticated region instead of chunk size
2. Make it clear that this configuration option only applies to upload + encrypting the blob and does not directly change the authentication region length used to download + decrypt the blob
3. Include a note that this configuration only applies to V2 encryption